src/node_url.cc

```diff
@@ -78,7 +81,7 @@ class URLHost {
   union Value {
     std::string domain_or_opaque;
     uint32_t ipv4;
-    uint16_t ipv6[8];
+    uint16_t ipv6[NS_IN6ADDRSZ / NS_INT16SZ];
```

Comment:
I think just `8` was a bit clearer here, to be honest…

Reply:
I just referred from https://sourceware.org/git/?p=glibc.git;a=blob;f=resolv/inet_ntop.c;h=c4d38c0f951013e51a4fc6eaa8a9b82e146abe5a;hb=HEAD#l119.

Reply:
I don’t think glibc is the gold standard for code readability here.

Reply:
I've changed it back.🐣